### ISSUE TITLE
add 'million' to chart  / remove tooltip

### ISSUE THREAD

**Issue by EmmyBruynseels** [oSoc19/bebudget#18]

**Comment by BramBleys** [comment]
Here's a way to do conversions in Javascript:
```

function fnum(x) {
	if(isNaN(x)) return x;

	if(x < 9999) {
		return x;
	}

	if(x < 1000000) {
		return Math.round(x/1000) + "K";
	}
	if( x < 10000000) {
		return (x/1000000).toFixed(2) + "M";
	}

	if(x < 1000000000) {
		return Math.round((x/1000000)) + "M";
	}

	if(x < 1000000000000) {
		return Math.round((x/1000000000)) + "B";
	}

	return "1T+";
}
```

**Comment by EmmyBruynseels** [comment]
fixed in 11a0a84 

Still need to remove the tooltip by clicking on the graph
